app/lib/actions/katello/repository/add_export_distributor.rb

```diff
@@ -0,0 +1,16 @@
+module Actions
+  module Katello
+    module Repository
+      class AddExportDistributor < Actions::Base
+        def plan(repo)
+          ::User.current = ::User.anonymous_admin
+          repo = ::Katello::Repository.find(repo.id)
+          plan_action(Pulp::Repository::AssociateDistributor,
+                      repo_id: repo[:pulp_id],
+                      config: {:http => true, :https => true},
```

Comment:
should those be false/false?   (it looks like below at the repo create action you are setting them as false?)